### PR TITLE
vdk: update contributing.md

### PR DESCRIPTION
**Pull request by tozka**
Clarify the current status around PR using forks so new users are not getting confused.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -35,7 +35,8 @@ We welcome many different types of contributions and not all of them need a Pull
 This is a rough outline of what a contributor's workflow looks like:
 
 * Make a fork of the repository within your GitHub account
-* Create a topic branch in your fork from where you want to base your work - name your branch according to our naming convention - person/<github-username>/<feature-name>
+   * Currently we accept contribution from forks only for examples and documentation changes until PR 854 is fixed. Until then please request write priveleges and create a branch in the main repo.
```
Comment (ivakoleva) [inline CONTRIBUTING.md:38]: I had to go and read the PR 854, to understand this statement.
Maybe first the guideline, then the explanation?
`Use forks to only contribute changes on examples and documentation. Currently we accept..`
And emphasise those changes are manually applied, is that correct?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -35,7 +35,8 @@ We welcome many different types of contributions and not all of them need a Pull
...
+* Create a topic branch from where you want to base your work - name your branch according to our naming convention - person/<github-username>/<feature-name>
```
Comment (ivakoleva) [inline CONTRIBUTING.md:39]: Move `Create a topic branch` as primary way of contributing, above the fork statement.

**Review by ivakoleva — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (tozka) [reply CONTRIBUTING.md:38]: OK

File: CONTRIBUTING.md (hunk 2 above)
Comment (tozka) [reply CONTRIBUTING.md:39]: Since we can accept some PRs using forks , Creating a fork for that case it is still a first step.

**Review by dakodakov — APPROVED** [review]

**Pull request merged**